modules/android/pages/working-with-vector-search.adoc

```diff
@@ -233,32 +232,133 @@ include::example$kotlin_snippets/app/src/main/kotlin/com/couchbase/codesnippets/
 NOTE: You can use less storage by using the `prediction()` function as the encoded vectors will only be stored in the index. 
 However, the index time will be longer as vector embedding generation is occurring at run time.
 
-== Vector SQL++ Functions
+== Create a Lazy Vector Index
 
-Couchbase Lite currently supports two SQL++ functions, `vector_match()` and `vector_distance()`.
+Lazy indexing is an alternate approach to using the standard predictive model with regular vector indexes which handle the indexing process automatically.
+You can use lazy indexing to use a ML model that is not available locally on the device and to create vector indexes without having vector embeddings in the documents.
 
-=== `vector_match(vectorIndexIdentifier, targetVectorExpr, [limit = 3])`
+[tabs]
+=====
 
-[cols = "3*"]
-|===
-|Parameter |Is Required |Description
+Java::
++
+--
+[source, java]
+----
 
-|vectorIndexIdentifier
-|Required
-|The name of the vector index to perform the vector search on.
-|targetVectorExpr
-|Required
-|The target vector expression that returns a vector in the form of an array of numbers.
-|limit
-|Optional
-|The limit number of the returned matched results.
-The maximum number allowed is 10000. An error will be returned when creating a query with a limit greater than 10000.
+include::java:example$snippets/common/main/java/com/couchbase/codesnippets/VectorSearchExamples.java[tags=vs-lazy-index-config, indent=2]
 
-|===
+----
+
+You can enable lazy vector indexing by setting the `isLazy` property to `true` in your vector index configuration.
+
+--
+
+
+Kotlin::
++
+--
+[source, kotlin]
+----
+
+include::example$kotlin_snippets/app/src/main/kotlin/com/couchbase/codesnippets/VectorSearchExamples.kt[tags=vs-lazy-index-config,indent=2]
+
+----
+
+You can enable lazy vector indexing by setting the `isLazy` property to `true` in your vector index configuration.
+
+--
+=====
+
+NOTE: Lazy Vector Indexing is opt in functionality, the `isLazy` property is set to `false` by default.
```

Comment:
```suggestion
NOTE: Lazy Vector Indexing is opt-in functionality, the `isLazy` property is set to `false` by default.
```

Reply:
Good spot, hyphenation is always a sticking point for me